### DISCUSSION TITLE
ALLDOCUBE M5X KERNEL 3.18 to 4.4

### DISCUSSION THREAD

**Discussion by Chihacheol** [community/community#21413] [Other Feature Feedback, Questions, & Ideas]
<p>How do I upgrade the kernel on the ALLDOCUBE M5X from 3.18 to <strong>4.4</strong>? Project Treble is supported, but does not boot to GSI.</p>

**Comment by ferdnyc** [comment, marked as answer]
<p>I’m not sure why you would expect the GitHub Community Forum to be able to answer that question (an OS support issue for a specific piece of tablet hardware), but the manufacturer’s community forum is located at: <a href="https://www.alldocube.com/en/community/" class="inline-onebox" rel="noopener nofollow ugc">Community – Alldocube Official Site</a></p>
<p>I can’t promise you’ll get an answer <em>there</em>, either, but your chances are roughly 1000x better there than here.</p>
